staging/src/k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1/constants_windows.go

```diff
@@ -0,0 +1,28 @@
+// +build windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+const (
+	// DevicePluginPath is the folder the Device Plugin is expecting sockets to be on
+	// Only privileged pods have access to this path
+	// Note: Placeholder until we find a "standard path"
+	DevicePluginPath = "c:\\var\\lib\\kubelet\\device-plugins\\"
```

Comment:
we had PRs in the past where having separate default paths for different OSes was undesired.

Reply:
Thanks for reviewing! 

> we had PRs in the past where having separate default paths for different OSes was undesired.

The question seems to be how to do it otherwise, since paths seems to be unavoidably different on Windows. 
The current approach was approved by @ddebroy from sig-windows. In his review on Feb 14 he requested:

> Is it possible to separate out only the `DevicePluginPath` const into the constants_windows.go and keep the rest of the constants in constants.go?

Which was then done by @aarnaud on Feb 17, and thumb-up approved by @ddebroy.

Reply:
I removed the separate default paths for windows to let Go and kubelet manage path on windows.
@thomacos You can test this new build https://github.com/aarnaud/k8s-directx-device-plugin/releases/download/0.1.0/kubelet-2020-07-08.exe

Reply:
@neolit123 it was for this reason I changed the path on windows with c:
https://github.com/kubernetes/kubernetes/issues/93262